src/ConfigFile.c

```diff
@@ -117,6 +117,9 @@ Configuration_Item Ros_ConfigFile_Items[] =
     { "stop_motion_on_disconnect", &g_nodeConfigSettings.stop_motion_on_disconnect, Value_Bool },
     { "inform_job_name", &g_nodeConfigSettings.inform_job_name, Value_String },
     { "allow_custom_inform_job", &g_nodeConfigSettings.allow_custom_inform_job, Value_Bool },
+    { "userlan_monitor_enabled", &g_nodeConfigSettings.userlan_monitor_enabled, Value_Bool },
+    { "userlan_monitor_port", &g_nodeConfigSettings.userlan_monitor_port, Value_Int },
+    { "userlan_monitor_max_num_drops", &g_nodeConfigSettings.userlan_monitor_max_num_drops, Value_Int },
```

Comment:
The current implementation does not include any sort of "debouncing logic", so this setting is currently not used.

Personally, I'd like to be notified of *any* drops, but I could be persuaded to allow configuring a specific threshold (ie: nr of drops) after which the disconnection logic should kick in.

(although perhaps a maximum amount of time for a link to remain down would make more sense)

Reply:
Do you feel a 'flaky' link would be something we should tolerate?

Reply:
I'm going to assume hw debouncing is sufficient for now.

We can always add another layer ourselves, but all my testing to far shows whenever the connection is unstable, it's really either the cable, connector or "the other side" (ie: switch, PC, connector on that side).